src/database/seeders/20210708041034-mock-data-streams.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return await queryInterface.bulkInsert('Streams', [
+      {
+        name: 'NIS',
+        createdAt: new Date(),
+        updatedAt: new Date(),
```

Comment:
Лучше добавить фиксированно id
```suggestion
      {
         id:1,
        name: 'NIS',
        createdAt: new Date(),
        updatedAt: new Date(),
```

Reply:
id автоматически добавляется при записи в бд